### PR TITLE
(GH-1056) Add support for automatic decompression to Get-WebFile

### LINKED ISSUE #1056: Files downloaded using Install-ChocolateyVsixPackage are packed in Zip file
### What You Are Seeing?
Vsix file downloaded using `Install-ChocolateyVsixPackage` is zipped, which results in checksum validation error.

### What is Expected?
File should be downloaded as Vsix, without being packed in a Zip file.

### How Did You Get This To Happen? (Steps to Reproduce)
Try installing [this package](https://chocolatey.org/packages/coderush-vs2015/16.1.9).
Source code [here](https://github.com/pascalberger/chocolatey-packages/blob/master/automatic/coderush-vs2015/tools/ChocolateyInstall.ps1)

### PR DESCRIPTION
**Pull request by pascalberger**
Add support for decompression to `Get-WebFile`.

Fixes #1056

### CODE REVIEW

File: src/chocolatey.resources/helpers/functions/Get-WebFile.ps1 [hunk 1]
```diff
@@ -159,6 +159,7 @@ param(
   $req.AllowAutoRedirect = $true
   $req.MaximumAutomaticRedirections = 20
   #$req.KeepAlive = $true
+  $req.AutomaticDecompression = [System.Net.DecompressionMethods]::GZip -bor [System.Net.DecompressionMethods]::Deflate
```
Comment (ferventcoder) [inline src/chocolatey.resources/helpers/functions/Get-WebFile.ps1]: Does this pick up both?

Comment (pascalberger) [inline src/chocolatey.resources/helpers/functions/Get-WebFile.ps1]: Yes it should. [DecompressionMethods](https://msdn.microsoft.com/en-us/library/system.net.decompressionmethods(v=vs.110).aspx) are flags.

Comment (ferventcoder) [inline src/chocolatey.resources/helpers/functions/Get-WebFile.ps1]: And this is helpful - works in .net 2+ https://msdn.microsoft.com/en-us/library/system.net.decompressionmethods(v=vs.80).aspx

**Comment by ferventcoder** [comment]
Merged into stable at https://github.com/chocolatey/choco/commit/bd15226fc81cce58eaafa80ab4ab145a1ca14613

**Pull request closed without merging**